src/main/java/com/kodilla/ecommercee/service/ProductGroupService.java

```diff
@@ -1,4 +1,28 @@
 package com.kodilla.ecommercee.service;
 
+import com.kodilla.ecommercee.dao.ProductGroupDao;
+import com.kodilla.ecommercee.domain.ProductGroup;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import javax.transaction.Transactional;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@Transactional
 public class ProductGroupService {
+    @Autowired
```

Comment:
Tutaj trzeba mapowa z/do Dto

Reply:
Nie wiem czy ten task jest tak naprawde jeszcze potrzebny, bo jestesmy juz na etapie implementacji kontrolerow i powiazanych z nimi maperr i service,wiec po czesci jest to juz zrobione na indywidualnych branch